simple-data-backend/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022,2024: Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2021,2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0. *
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# * SPDX-License-Identifier: Apache-2.0
+
+# Dependencies
+FROM maven:3-eclipse-temurin-17-alpine AS maven
+ARG BUILD_TARGET=simple-data-backend
+
+WORKDIR /build
+
+COPY . .
+
+# the --mount option requires BuildKit.
+RUN --mount=type=cache,target=/root/.m2 mvn -B clean package -am -DskipTests
+
+
+# Copy the jar and build image
+FROM eclipse-temurin:17-jre-alpine AS simple-data-backend
```

Comment:
https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=17-jre-alpine isn't available arm64 cpu architecture, would it maybe be possible to use another image? https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=jre-alpine

Reply:
We have a TRG for our allowed base images: [eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02#aligned-base-images](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02#aligned-base-images)

Reply:
Only `21-jre-alpine` is provided as an arm image. @ds-jhartmann could we use it?

Reply:
Yes, I switched to Java 21 and added the multi platform build